### PR TITLE
Remaned devlab/jenkins/run_functional_tests.sh -> devlab/jenkins/setup_lab.sh

### PR DESCRIPTION
**Pull request by andybondar**

### CODE REVIEW

File: devlab/jenkins/run_functional_tests.sh [hunk 1]
```diff
@@ -43,12 +39,11 @@ pip install pip==6.1.1
...
 
 echo "Preparing lab"
-
 pushd devlab
 echo 'Removing old VMs if exist, first try with vagrant...'
 vagrant destroy --force
-echo 'then with vboxmanage...'
-for v in `vboxmanage list vms | awk '{print $1}'`; do vboxmanage unregistervm --delete $v; done
+echo 'then unregister VMs...'
+vboxmanage list vms | awk -F"{" '{print $2}' | awk -F"}" '{print $1}' | xargs -I {} vboxmanage unregistervm --delete {}
```
Comment (realsystem) [inline devlab/jenkins/run_functional_tests.sh]: Please, don't remove for and use sed:

for vm in `vboxmanage list vms | awk '{print $2}' | sed 's/^{*//;s/}*$//'`; do vboxmanage unregistervm --delete $vm; done

Comment (andybondar) [inline devlab/jenkins/run_functional_tests.sh]: Done

**Comment by realsystem** [comment]
+1

File: devlab/jenkins/run_functional_tests.sh [hunk 2]
```diff
@@ -2,36 +2,31 @@
...
-JOB_NAME="${JOB_NAME:-cloudferry-functional-tests}"
-
-trap 'clean_exit $LINENO $BASH_COMMAND; exit' SIGHUP SIGINT SIGQUIT SIGTERM EXIT
-clean_exit()
-{
-    pushd ${CF_DIR}/devlab
-    vboxmanage list vms
-    vagrant destroy --force
-    popd
-}
```
Comment (realsystem) [inline devlab/jenkins/run_functional_tests.sh]: Please, don't do this. Just keep it in the script and use copy in the job if needed.

File: devlab/jenkins/run_functional_tests.sh [hunk 3]
```diff
@@ -2,36 +2,31 @@
...
+export JOB_NAME="${JOB_NAME:-cloudferry-functional-tests}"
+
+#trap 'clean_exit $LINENO $BASH_COMMAND; exit' SIGHUP SIGINT SIGQUIT SIGTERM EXIT
+#clean_exit()
+#{
+#    pushd ${CF_DIR}/devlab
+#    vboxmanage list vms
+#    vagrant destroy --force
+#    popd
+#}
```
Comment (roman-verchikov) [inline devlab/jenkins/run_functional_tests.sh]: Is it a debug leftover? Keeping commented code is a bad practice, also I believe script should handle any error properly, which means we should keep this code uncommented.

**Comment by realsystem** [comment]
+1

**Comment by roman-verchikov** [comment]
+1

**Pull request merged**